src/main/java/dev/j3fftw/litexpansion/items/PortableCharger.java

```diff
@@ -38,15 +38,15 @@
  */
 public class PortableCharger extends SimpleSlimefunItem<ItemUseHandler> implements Listener, Rechargeable {
 
-    private final int[] border = { 5, 6, 7, 14, 16, 23, 24, 25 };
-    private final int powerSlot = 11;
-    private final int chargeSlot = 15;
-    private final int chargeSpeed = 50;
+    private final int[] BORDER = { 5, 6, 7, 14, 16, 23, 24, 25 };
+    private final int POWER_SLOT = 11;
+    private final int CHARGE_SLOT = 15;
+    private final float CHARGE_SPEED = 50;
     private final Plugin plugin = LiteXpansion.getInstance();
 
     public PortableCharger() {
         super(Items.LITEXPANSION, Items.PORTABLE_CHARGER, RecipeType.ENHANCED_CRAFTING_TABLE, new ItemStack[] {
-            Items.COPPER_CABLE, Items.REFINED_IRON, Items.COPPER_CABLE,
+            Items.REFINED_IRON, Items.COPPER_CABLE, Items.REFINED_IRON,
```

Comment:
Its still on the outside...

Reply:
its the power port :c originally it was meant to be the positive and negative nodes but that didnt really make sense logically